app/database/scripts/move-location-images.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-console */
+
+import { v4 as uuid } from "uuid";
+import { getSupabaseAdmin } from "~/integrations/supabase/client";
+import { PUBLIC_BUCKET } from "~/utils/constants";
+import { db } from "../db.server";
+
+/**
+ * This script moves the location images from database to the supabase storage.
+ * To move the images, we have the following steps:
+ * 1. Get all the locations from the database which have images
+ * 2. Move the images from the database to the supabase storage
+ * 3. Update the url of image in the database in Location table
+ *
+ * Images are going to be stored in following format:
+ * files/organizationId/locations/locationId/imageId
+ */
+(async function moveLocationImages() {
```

Comment:
Okey a few things here:

1. How do we want to run this? I think running it from terminal could be problematic, because of being able to connect to the correct database as well as failures. I think we should consider making this into a function that the super admin can run from the admin dashboard.
2. How do we make sure this doesnt time out. We have around 1000 locations with images in our app. I think this will fail and we have to consider doing it in batches or something like this.